integrationtests/k8s_test.go

```diff
@@ -200,7 +211,23 @@ func TestK8s(t *testing.T) {
 		assert.NoError(t, err)
 		defer terraform.Destroy(t, terraformOptions)
 
-		AssertNodesAreReady(t, terraformOptions, privateKey)
+		// Check that the outputs not empty
+		masterIP := terraform.Output(t, terraformOptions, "ygg_ip")
+		assert.NotEmpty(t, masterIP)
+
+		// Check that master and workers is reachable
+		// testing connection on port 22, waits at max 3mins until it becomes ready otherwise it fails
+		ok = TestConnection(masterIP, "22")
+		assert.True(t, ok)
+
+		ticker := time.NewTicker(2 * time.Second)
+		for now := time.Now(); time.Since(now) < 1*time.Minute; {
+			<-ticker.C
+			output, err := RemoteRun("root", masterIP, "zinit list", privateKey)
```

Comment:
this checks if the k3s service is running on the master node, which i believe does not say anything about the other nodes in the cluster. imo it should not be the test's purpose to check if the flist works, but i also think that this it's just easier to check if the cluster works or not.

Reply:
yes @khaledyoussef24 plz use kubectl get nodes instead and make sure all workers are running